### PR TITLE
Update readme.html

### PR DESCRIPTION
**Pull request by hitjethva**
Add YOURLS installation link for Ubuntu 20.04 version.

### CODE REVIEW

**Review by LeoColomb — APPROVED** [review]
LGTM 👍 
Thx @hitjethva!

Side note: @hitjethva is also the author of the post.

**Comment by hitjethva** [comment]
Please merge my pull request

File: readme.html [hunk 1]
```diff
@@ -297,6 +297,7 @@ <h1><a href="https://yourls.org/">YOURLS: <span>Y</span>our <span>O</span>wn <sp
             <p>The <a href="https://yourls.org/wiki">wiki</a> is full of important information. Regarding installation, you'll find help for Apache, IIS and Nginx</p>
             <p>Here is a list of some (unofficial) install guides using automated tools:</p>
 			<ul>
+                <li><a href="https://linuxbuz.com/linuxhowto/install-yourls-ubuntu-20-04">How to Install YOURLS URL Shortener on Ubuntu 20.04</a></li>
 				<li><a href="https://www.rosehosting.com/blog/how-to-install-yourls-on-ubuntu-14-04/">How to Install YOURLS on Ubuntu 14.04</a></li>
 				<li><a href="https://computingforgeeks.com/how-to-install-yourls-your-own-url-shortener-on-ubuntu-18-04-bionic-beaver-linux/">How to install YOURLS on Ubuntu 18.04</a></li>
```
Comment (dgw) [inline readme.html:302]:
```suggestion
				<li><a href="https://www.rosehosting.com/blog/how-to-install-yourls-on-ubuntu-14-04/">How to Install YOURLS on Ubuntu 14.04</a></li>
				<li><a href="https://computingforgeeks.com/how-to-install-yourls-your-own-url-shortener-on-ubuntu-18-04-bionic-beaver-linux/">How to install YOURLS on Ubuntu 18.04</a></li>
                <li><a href="https://linuxbuz.com/linuxhowto/install-yourls-ubuntu-20-04">How to Install YOURLS URL Shortener on Ubuntu 20.04</a></li>
```

**Review by dgw — CHANGES_REQUESTED** [review]
I thought about bowing to the impatience, but the ordering of these list items bugs me.

File: readme.html (hunk 1 above)
Comment (hitjethva) [reply readme.html:302]: So how can I fix this?

**Pull request merged**